src/rastervision/evaluations/object_detection_evaluation.py

```diff
@@ -29,15 +27,31 @@ def compute_od_eval(ground_truth_labels, prediction_labels):
 
 def parse_od_eval(od_eval, class_map):
     class_to_eval_item = {}
+
+    score_ind = -1
     for class_id in range(1, len(class_map) + 1):
-        class_name = class_map.get_by_id(class_id).name
         gt_count = int(od_eval.num_gt_instances_per_class[class_id - 1])
+        class_name = class_map.get_by_id(class_id).name
 
-        # If there are predictions for this class.
-        if len(od_eval.precisions_per_class[class_id - 1]) > 0:
-            precisions = od_eval.precisions_per_class[class_id - 1]
-            recalls = od_eval.recalls_per_class[class_id - 1]
-            # Get precision and recall assuming all predicted boxes are used.
+        if gt_count == 0:
+            eval_item = EvaluationItem(
+                class_id=class_id, class_name=class_name)
+        else:
+            # precisions_per_class has an element appended to it for each
+            # class_id that has gt_count > 0. This means that the length of
+            # precision_per_class can be shorter than the total number of
+            # classes in the class_map. Therefore, we use score_ind to index
+            # into precisions_per_class instead of simply using class_id - 1.
+            score_ind += 1
+
+            # Precisions and recalls across a range of detection thresholds.
+            precisions = od_eval.precisions_per_class[score_ind]
+            recalls = od_eval.recalls_per_class[score_ind]
+
+            # If we use the lowest detection threshold (ie. use all detected
```

Comment:
to get this to run with xview i had to change this only run the code below if precisions and recalls were not empty arrays

Reply:
You can add a commit to this one, and I'll take a look at it.

Reply:
ok, just did